### PR TITLE
Clarified 404.html usage

### PR DESCRIPTION
**Pull request by rapilabs**
Thanks to @kezabelle for reviewing and making more explicit.

### CODE REVIEW

File: docs/topics/http/views.txt [hunk 1]
```diff
@@ -122,14 +122,14 @@ Example usage::
...
-In order to use the ``Http404`` exception to its fullest, you should create a
-template that is displayed when a 404 error is raised. This template should be
-called ``404.html`` and located in the top level of your template tree.
-
-If you provide a message when raising an ``Http404`` exception, it will appear
-in the standard 404 template displayed when :setting:`DEBUG` is ``True``. Use
-these messages for debugging purposes; they generally aren't suitable for use
-in a production 404 template.
+In order to show customized HTML when a 404 is encountered, you can create a
+an HTML template named ``404.html`` and place it in the top level of your
```
Comment (funkybob) [inline docs/topics/http/views.txt]: "a an"

File: docs/topics/http/views.txt [hunk 2]
```diff
@@ -122,14 +122,14 @@ Example usage::
...
+template tree.  This template will then be served when :setting:`DEBUG` is set
+to ``False``.
+
+While :setting:`DEBUG` is ``True``, messages passed to ``Http404`` will appear
```
Comment (funkybob) [inline docs/topics/http/views.txt]: Is that the proper terminology for arguments passed when raising an exception?  "messages passed to"?

**Comment by rapilabs** [comment]
@funkybob the "messages passed to" was more an attempt to keep in line with the existing phrasing, also to be more descriptive to non-programmers.  But you're right, it's better to keep it technically correct :)

File: docs/topics/http/views.txt [hunk 3]
```diff
@@ -122,14 +122,14 @@ Example usage::
...
 
-In order to use the ``Http404`` exception to its fullest, you should create a
-template that is displayed when a 404 error is raised. This template should be
-called ``404.html`` and located in the top level of your template tree.
-
-If you provide a message when raising an ``Http404`` exception, it will appear
-in the standard 404 template displayed when :setting:`DEBUG` is ``True``. Use
-these messages for debugging purposes; they generally aren't suitable for use
-in a production 404 template.
+In order to show customized HTML when a 404 is encountered, you can create a
```
Comment (timgraham) [inline docs/topics/http/views.txt]: "when Django returns a 404"?
an HTML

File: docs/topics/http/views.txt [hunk 4]
```diff
@@ -122,14 +122,14 @@ Example usage::
...
+HTML template named ``404.html`` and place it in the top level of your
+template tree.  This template will then be served when :setting:`DEBUG` is set
```
Comment (timgraham) [inline docs/topics/http/views.txt]: one space between sentences please

File: docs/topics/http/views.txt [hunk 5]
```diff
@@ -122,14 +122,14 @@ Example usage::
...
+to ``False``.
+
+While :setting:`DEBUG` is ``True``, arguments passed to ``Http404`` will appear
```
Comment (timgraham) [inline docs/topics/http/views.txt]: While ->When
"arguments" -> you can provide a message to Http404 and it will appear...

**Comment by timgraham** [comment]
Please squash commits when updating and phrase commit message in past tense as described in the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist). Thanks!

File: docs/topics/http/views.txt (hunk 3 above)
Comment (rapilabs) [inline docs/topics/http/views.txt]: Actually I had "when Django returns a 404" and @kezabelle suggested using "encountered".  I don't mind either way, thought I should just mention that first though :)

File: docs/topics/http/views.txt [hunk 6]
```diff
@@ -122,14 +122,15 @@ Example usage::
...
-in the standard 404 template displayed when :setting:`DEBUG` is ``True``. Use
-these messages for debugging purposes; they generally aren't suitable for use
-in a production 404 template.
+In order to show customized HTML when Django returns a 404, you can create an
+HTML template named ``404.html`` and place it in the top level of your
+template tree. This template will then be served when :setting:`DEBUG` is set
+to ``False``.
+
+When :setting:`DEBUG` is ``True``, you can provide a message to ``Http404`` and
+it will appear in the standard 404 debug template.  Use these messages for
```
Comment (MarkusH) [inline docs/topics/http/views.txt]: double white space

**Comment by MarkusH** [comment]
Merged in 7a98442f9645102601c5a18b2ff93b7433e3a01d. Thanks! :cookie:

**Pull request closed without merging**